modules/mod_auth2fa/templates/_auth2fa_html_body.tpl

```diff
@@ -1,5 +1,7 @@
-{% with m.auth2fa.user_mode as mode %}
-    {% if mode == 2 or (mode == 1 and not m.auth2fa.is_totp_requested) %}
-        {% wire postback={request_2fa} delegate=`mod_auth2fa` %}
-    {% endif %}
-{% endwith %}
+{% if not m.auth2fa.is_totp_enabled %}
```

Comment:
Why should this only happen if totp is not enabled?

Reply:
Not enabled for this user.  If the user as TOTP enabled then we don't need to ask.